code/TestReactFormBuilder.php

```diff
@@ -64,4 +69,22 @@ public function getEditForm($id = null, $fields = null) {
 
         return Form::create($this, 'TestEditForm', FieldList::create(), FieldList::create());
     }
+
+
+    public function providePermissions()
+    {
+        $code = static::getRequiredPermissions();
+        $title = LeftAndMain::menu_title(static::class);
+        return [
+            $code => [
+                // Item in permission selection identifying the admin section. Example: Access to 'Files & Images'
+                'name' => _t(
+                    static::class . '.ACCESS',
+                    "Access to '{title}' section",
+                    ['title' => $title]
+                ),
+                'category' => _t(static::class . '.CMS_ACCESS_CATEGORY', 'CMS Access')
+            ]
+        ];
+    }
```

Comment:
Basically a straight copy/paste from `LeftAndMain` which handles this for model admins